src/sage/categories/modules_with_basis.py

```diff
@@ -2062,17 +2063,26 @@ def trailing_term(self, *args, **kwds):
             """
             return self.parent().term(*self.trailing_item(*args, **kwds))
 
-        def map_coefficients(self, f):
+        def map_coefficients(self, f, new_base_ring=None):
             """
-            Mapping a function on coefficients.
+            Return the element obtained by applying ``f`` to the non-zero
+            coefficients of ``self``.
+
```

Comment:
This method will need clarification because of:
- #37526

Reply:
Could you please elaborate?  I don't see a change to `map_coefficients` in #37526, nor a deprecation.

Reply:
ping?

Reply:
Sorry for the delay. I was concerned regarding the meaning of "coefficients" when submodules are involved.
The code looks fine -- as it's just mapping monomial coefficients to monomial coefficients.